gpu/gpu_darwin.go

```diff
@@ -64,3 +64,8 @@ func (l GpuInfoList) GetVisibleDevicesEnv() (string, string) {
 	// No-op on darwin
 	return "", ""
 }
+
+// Unused on Darwin
+func GetDepDir() string {
```

Comment:
I think we can omit this – the only use of `GetDepDir` seems to be in `gpu.go` which is `linux || windows`